### PR TITLE
tests for approver apis, code coverage 100% for the same

### PR DESCRIPTION
**Pull request by harshithaputtaswamy**

### CODE REVIEW

File: test/integration/approver/test_approver_apis.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+  if approve_report:
+    assert dict_compare_keys(approve_report["data"], mock_reports[0]) == [], 'fyle.v1beta.approver.reports.list() has stuff that mock_data doesnt'
+    assert dict_compare_keys(mock_reports[0], approve_report["data"]) == [], 'mock_data.reports.get() has stuff that fyle doesnt'
+
+
+def test_get_by_id(fyle, mock_data):
+  reports_generator = fyle.v1beta.approver.reports.get_by_id(id_="rp6yFhpzfEhc")
+  mock_reports = mock_data.reports.get()
+  print(reports_generator)
```
Comment (Sravanksk) [inline test/integration/approver/test_approver_apis.py:66]: remove print statement

File: test/integration/approver/utilities.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  mock_data_dict = get_mock_data_dict(filename)
+  mock_data = Mock()
+  mock_data.categories.get.return_value = mock_data_dict['categories']
+  mock_data.reports.get.return_value = mock_data_dict['reports']
+
+  return mock_data
+
+
+def get_mock_data():
+  return get_mock_data_from_file('mock_approver_data.json')
```
Comment (Sravanksk) [inline test/integration/approver/utilities.py:23]: add a new line at the end of the file

**Review by Sravanksk — APPROVED** [review]

File: test/integration/approver/utilities.py (hunk 2 above)
Comment (harshithaputtaswamy) [reply test/integration/approver/utilities.py]: done

File: test/integration/approver/test_approver_apis.py (hunk 1 above)
Comment (harshithaputtaswamy) [reply test/integration/approver/test_approver_apis.py]: done

File: test/integration/approver/conftest.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture
+def mock_data():
+  return get_mock_data()
+
+@pytest.fixture(scope='module')
+def fyle():
+  return fyle_connect()
```
[the remark this answers is not in the document]
Comment (harshithaputtaswamy) [reply test/integration/approver/conftest.py]: done

**Pull request merged**